tests/FluentTest.php

```diff
@@ -913,12 +913,12 @@ public function testWriteWithLocale()
         $item2->Title = 'English 2';
         $item2->write();
 
-        // check alternate locale title unchanged
+        // check alternate locale title reflects the default locale value
         $es2Title = Fluent::with_locale('es_ES', function () use ($item2ID) {
             $item2 = FluentTest_TranslatedObject::get()->byId($item2ID);
             return $item2->Title;
         });
-        $this->assertEquals($es2Title, 'Spanish 2');
+        $this->assertEquals($es2Title, 'English 2');
```

Comment:
I would say, maybe we want to disable this behaviour until AFTER the record has been saved at least once in the default locale, right?